Formula/pyoxidizer.rb

```diff
@@ -0,0 +1,23 @@
+class Pyoxidizer < Formula
+  desc "Modern Python application packaging and distribution tool"
+  homepage "https://github.com/indygreg/PyOxidizer"
+  url "https://github.com/indygreg/PyOxidizer/archive/refs/tags/pyoxidizer/0.17.tar.gz"
```

Comment:
I think preference is shorter GitHub URL:
```suggestion
  url "https://github.com/indygreg/PyOxidizer/archive/pyoxidizer/0.17.tar.gz"
```